src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Patterns.cs

```diff
@@ -106,7 +106,7 @@ private BoundExpression CompareWithConstant(BoundExpression input, BoundExpressi
                 );
         }
 
-        BoundExpression MakeDeclarationPattern(SyntaxNode syntax, BoundExpression input, LocalSymbol target, bool requiresNullTest)
+        BoundExpression MakeDeclarationPattern(SyntaxNode syntax, BoundExpression input, BoundExpression target, bool requiresNullTest)
```

Comment:
It isn't. Normally `target` is a reference to the local, but the EE rewriter changes it to something else.

---
In reply to: [77712197](https://github.com/dotnet/roslyn/pull/13535#discussion_r77712197) [](ancestors = 77712197)